flink_scrat/utils.py

```diff
@@ -30,3 +31,47 @@ def setup_logging():
         logging.config.dictConfig(config)
     else:
         logging.basicConfig(level=default_level)
+
+def handle_response(req_response):
+        req_response.raise_for_status()
+        return req_response.json()
+
+def find_manager_address(yarn_address, yarn_port, app_name):
```

Comment:
Why didn't you create a `YarnConnector` class or something like that?

Reply:
Be cause all it does is finding the params that will be used to instantiate the `JobManagerConnector`

Reply:
It could be in another file, but I don't find it should be a class by it own